### PR TITLE
Fluentd: explicitly set refresh_interval to 1s instead of 60s by default.

### PR DESCRIPTION
**Pull request by Gerhut**
This change will fix log missing if the job is finished in less than 60 seconds.

May affect I/O performance.

### CODE REVIEW

File: src/ClusterBootstrap/services/logging/fluentd-config.yaml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    <filter kubernetes.**>
+      @type kubernetes_metadata
+      @id filter_kube_metadata
+    </filter>
+  prometheus.conf: |
+    # Prometheus metric exposed on 0.0.0.0:24231/metrics
+    <source>
+      @type prometheus
+      bind "#{ENV['FLUENTD_PROMETHEUS_BIND'] || '0.0.0.0'}"
+      port "#{ENV['FLUENTD_PROMETHEUS_PORT'] || '24231'}"
```
Comment (xudifsd) [inline src/ClusterBootstrap/services/logging/fluentd-config.yaml]: Is this the same to `{{cnf["fluentd"]["port"]}}`?

Comment (Gerhut) [reply src/ClusterBootstrap/services/logging/fluentd-config.yaml]: It's overwritten in envvar, which is `{{cnf["fluentd"]["port"]}}`

See [#diff-a40d28c47725cc72f15c36d7ae3f90eaR37-R38]()

**Comment by Anbang-Hu** [comment]
@Gerhut please help resolve the conflict and merge

**Comment by Gerhut** [comment]
#836 includes this, so this could be closed.

**Pull request closed without merging**